### PR TITLE
Fixed #638 -- Added django_usage field underneath description on corpo…

### LINKED ISSUE #638: Add "How does your organization use Django?" to corporate membership app.
It will help to cut down the "Anyone have any ideas on who these people are?" discussion that the board has when approving applications.

### PR DESCRIPTION
**Pull request by nanuxbe**
…rate memebership form

### CODE REVIEW

**Comment by bmispelon** [comment]
I think you're missing a migration file for the new field. Could you add it to the commit?

**Comment by nanuxbe** [comment]
You're right, sorry about that, I've added it :-)

**Comment by bmispelon** [comment]
Looks good to me code-wise. Is there someone on the board who could validate the feature?

File: members/models.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ class CorporateMember(models.Model):
     membership_level = models.IntegerField(choices=MEMBERSHIP_LEVELS)
     address = models.TextField(blank=True)
     notes = models.TextField(blank=True, help_text='Not displayed publicly.')
+    django_usage = models.TextField('How does your organization use Django?', blank=True,
```
Comment (MarkusH) [inline members/models.py]: Do you mind using hanging indents as above (lines 67-71) please. :smile:

Comment (MarkusH) [inline members/models.py]: Apart from that, why make it `blank=True` when `members/forms.py` has a `required=True` there?

Comment (timgraham) [inline members/models.py]: I guess that allows editing existing members in the admin without putting some value in the field.

**Comment by timgraham** [comment]
merged in 8f2751e00511cefb0ec41f6fb2824813ac674e68, thanks!

**Pull request closed without merging**